packages/shared/src/components/squads/SquadHeaderMenu.tsx

```diff
@@ -65,12 +66,10 @@ export default function SquadHeaderMenu({
   const items = useMemo(() => {
     const list: ContextMenuItemProps[] = [
       {
-        Icon: TourIcon,
-        onClick: () =>
-          openModal({
-            type: LazyModal.SquadTour,
-          }),
-        label: 'Learn how Squads work',
+        Icon: FeedbackIcon,
```

Comment:
Did you remove the tour in favor, not 100% sure that was intended (but could be wrong)
Please just make double sure they don't just want to add a option.

Reply:
Per design it should be removed but I'll check.